templates/10-k8s-master.conf.j2

```diff
@@ -1,25 +1,31 @@
 # Managed by Ansible (role: router-k8s)
 # DO NOT EDIT!
 
-frontend HTTP-intern
+{% for cluster in router_k8s_clusters %}
+frontend HTTP-intern-{{ cluster.name }}
     mode tcp
-    bind {{ kubernetes_ha_ip_intern }}:80
-    default_backend kube-apiserver
+    bind {{ cluster.ha_ip_intern }}:80
+    default_backend kube-apiserver-{{ cluster.name }}
 
-frontend HTTPS-intern
+frontend HTTPS-intern-{{ cluster.name }}
     mode tcp
-    bind {{ kubernetes_ha_ip_intern }}:443
+    bind {{ cluster.ha_ip_intern }}:443
     mode     tcp
     timeout client 2147483647
-    default_backend kube-apiserver
+    tcp-request inspect-delay 5s
```

Comment:
Why do SNI inspection if there is only one backend?

Reply:
This seems to be a relic of the old VIS setup where we still had the Tectonic backend (I have worked on top of the `vis` branch). It looks unnecessary indeed, I will remove it.